### PR TITLE
[GH-133]Add UnityPool.create/delete for storops

### PR DESCRIPTION
**Pull request by peter-wangxu**
Add create/delete methods for UnityPool object.

### CODE REVIEW

File: storops_test/unity/resource/test_pool.py [hunk 1]
```diff
@@ -141,6 +146,34 @@ def test_get_nested_resource_filter_by_non_id(self):
...
+            cli=cli, name='test_pool', description='Unity test pool.',
+            raid_groups=raid_groups, alert_threshold=15,
+            is_harvest_enabled=True, is_snap_harvest_enabled=True,
+            pool_harvest_high_threshold=80, pool_harvest_low_threshold=40,
+            snap_harvest_high_threshold=80, snap_harvest_low_threshold=40,
+            is_fast_cache_enabled=True, is_fastvp_enabled=True,
+            pool_type=PoolTypeEnum.DYNAMIC)
+        assert_that(pool.id, equal_to('pool_4'))
+        assert_that(pool.pool_type, equal_to(PoolTypeEnum.DYNAMIC))
+        assert_that(pool.is_all_flash, equal_to(False))
```
Comment (jealous) [inline storops_test/unity/resource/test_pool.py]: Add a case for pool name already exists.

File: storops_test/unity/resource/test_pool.py [hunk 2]
```diff
@@ -141,6 +146,34 @@ def test_get_nested_resource_filter_by_non_id(self):
...
+
+    @patch_rest
+    def test_delete_pool(self):
```
Comment (jealous) [inline storops_test/unity/resource/test_pool.py]: Add a case for pool not found.

File: storops_test/unity/resource/test_pool.py (hunk 1 above)
Comment (peter-wangxu) [inline storops_test/unity/resource/test_pool.py]: Done

Comment (peter-wangxu) [inline storops_test/unity/resource/test_pool.py]: Done

**Pull request merged**